### PR TITLE
added mocks and front end for last ten commits

### PR DESCRIPTION
**Pull request by Tristan-Autin**

### CODE REVIEW

File: src/pages/GitView.vue [hunk 1]
```diff
@@ -1,10 +1,53 @@
...
-export default {};
+import { useRoute } from 'vue-router';
+
+import commits from '../mocks/commits.js';
+
+export default {
+  setup() {
+    const route = useRoute();
+    const query = route.query;
+    const { folderPath } = query;
```
Comment (antoinechassagne) [inline src/pages/GitView.vue:35]: Why not directly `const { folderPath } = route.query;` ?

File: src/pages/GitView.vue [hunk 2]
```diff
@@ -1,10 +1,53 @@
...
+    <div class="flex justify-between py-4 px-4">
+      <router-link to="/" class="bg-green-500 px-5 py-2 text-white rounded"
+        >Back</router-link
+      >
+      <div>{{ folderPath }}</div>
+      <div></div>
+    </div>
+    <div class="w-full flex h-full">
+      <div class="flex self-center line relative">
+        <!-- <div class="line"></div> -->
```
Comment (antoinechassagne) [inline src/pages/GitView.vue:12]: Maybe we should remove this comment ?

**Review by clemcode — CHANGES_REQUESTED** [review]
I think that an acceptance criteria is not fullfiled : we have to specify the commit where the `HEAD` is, see #2

**Review by clemcode — APPROVED** [review]
👍

**Review by felixbouveret — APPROVED** [review]

**Pull request merged**